### DISCUSSION TITLE
Best way to contact regarding plugin development?

### DISCUSSION THREAD

**Discussion by itadmsa** [kimai/kimai#4338] [General] [answered]
Hello, I'm not certain if this is the correct way to go about doing this so please let me know if I need to do something differently.

The company I work for is needed to switch to a new time tracking app and Kimai looks to be the closest fit for what we need.

We have a few ideas that would greatly help with our workflow within Kimai and saw on the plugin store that the developer offers custom plugin development. After discussing with my boss and typing out the ideas in detail, we sent an email request to both support@kimai.org and info@keleo.de about 2 weeks ago.

I saw on one of the developer's other pages that he simply gets too many emails and it is easy for communications to get lost, so they also suggest posting on Github, so I made an account and am posting here.

Since we are interested in discussing paying for these features to be developed, I wasn't sure if I should just post them here as a feature request or not. None of them are anything that would need to be specific to our company.

Thank you

**Comment by kevinpapst** [comment, marked as answer]
As I do not know who you are, I can only guess that I saw your email and it is still pending in my inbox. If you are the one that sent me a mail with more than 1100 words and red headers, thank you. 

Not that I am trying to lecture you. But if you want to get help, you should start with a simple introduction. And then make it as simple for me as possible to move forward. I am not a big or small corporate but a single person and such a business runs different. I love seeing new users of Kimai, but you can't expect me to put such an amount of effort and time into your issues without prior contact 🤷

**Comment by itadmsa** [comment]
Unfortunately, that indeed was me. I have a tendency to overdo it, especially when busy which can be counter-productive.. I saw your email and have followed up there. Thank you for your response and the constructive feedback.
